device-types/Digi/Digi-CM-8.yml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: Digi
+model: Digi CM 8
+slug: digi-cm-8
+part_number: '70001931'
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (2.50 lbs or 1.10 kg)